CHANGELOG.md

```diff
@@ -4,8 +4,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
 ## [Unreleased]
 
+### Added
+- Stability improvements for CloudForClustomers
+- Added batching with top parameter in `CloudForCustomersReportToADLS`
+- Added support for Parquet in `CloudForCustomersReportToADLS`
+- Added channel filtering in `CloudForCustomersReportToADLS`
+
+### Changed
+- Unifying `CloudForCustomersToADLS` into `CloudForCustomersReportToADLS`
```

Comment:
Incorrect wording, not sure what "unifying" means here. Did you mean that you renamed it? Or changed the functionality from supporting collections only to reports only?